### PR TITLE
Disable hover effect on tooltips.

### PR DESCRIPTION
**Pull request by imizaac**
## About the PR 
Fixes https://github.com/goonstation/goonstation/issues/3218. Minor bugfix that will dramatically improve the usability of tooltips

## Why's this needed? 

Pretty simple. You should not be able to select text that comes from a `:hover` attribute. Doing so provides a horrible UX.

## Changelog

```
(u)BobertBrockson:
(+)You can no longer select tooltips generated by `:hover` or holding `alt+:hover`.
```

### CODE REVIEW

**Review by ZeWaka — APPROVED** [review]

**Comment by imizaac** [comment]
@ZeWaka Okay, cool so I have an approval. I wanna touch base around conventions here. 

Do I wait for N number of reviews, wait for all outstanding reviews, or something similar before merging?

Also do I press merge or would you prefer a maintainer with a higher level of access press merge?

Do we have a timing window in which we prefer to perform merges?

**Comment by ZeWaka** [comment]
> @ZeWaka Okay, cool so I have an approval. I wanna touch base around conventions here.

> ...

You can't merge.

**Comment by imizaac** [comment]
@ZeWaka Perfect! Even better.

**Pull request merged**
